db/migrate/20160226103946_add_location_to_shop.rb

```diff
@@ -0,0 +1,7 @@
+class AddLocationToShop < ActiveRecord::Migration
+  def change
+    add_column :shops, :longitude, :numeric
+    add_column :shops, :latitude, :numeric
+    add_index  :shops, [:lat, :lng]
```

Comment:
Sorry, I forgot to change the names of the columns in my copy-paste code above, but you need to change `lat` → `latitude` & `lng` → `longitude`.